src/components/Base/Table/index.scss

```diff
@@ -171,6 +171,8 @@
       transition: all $transition-speed;
       word-wrap: break-word;
       word-break: break-all;
+      font-size: 12px;
```

Comment:
`font-size: $size-small`

Reply:
General font not used variable，only color used variable